### PR TITLE
Add tox in ra1

### PR DESCRIPTION
**Pull request by collivier**
The RST verification is temporarily disabled to allow a full process to Jenkins.

### CODE REVIEW

File: doc/ref_arch/openstack/chapters/chapter02.rst [hunk 1]
```diff
@@ -119,8 +121,8 @@ Table 2-1a: Reference Model Requirements: Cloud Infrastructure Software
 Profile Capabilities
 
    **1** Defined in the ``.bronze`` configuration in `RM section 4.2.6
-   Storage
-   Extensions <../../../ref_model/chapters/chapter04.md#426-storage-extensions>`__\ 
+   Storage Extensions
+   <../../../ref_model/chapters/chapter04.md#426-storage-extensions>`__
```
Comment (karinesevilla) [inline doc/ref_arch/openstack/chapters/chapter02.rst:125]: @collivier Link not changed, it should be 
:ref:`ref_model/chapters/chapter04:storage extensions`

File: doc/ref_arch/openstack/chapters/chapter02.rst [hunk 2]
```diff
@@ -427,8 +435,10 @@ Cloud Infrastructure Hardware Profile Requirements (source `RM 5.4 <../../../ref
 Table 2-4a: Reference Model Requirements - Cloud Infrastructure Hardware
 Profile Requirements
 
-Cloud Infrastructure Hardware Profile-Extensions Requirements (source `RM 5.4 <../../../ref_model/chapters/chapter05.md#54-cloud-infrastructure-hardware-profiles-features-and-requirements>`__)
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Cloud Infrastructure Hardware Profile-Extensions Requirements
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+(source :ref:`ref_model/chapters/chapter05:cloud infrastructure software profiles features and requirements`)
```
Comment (karinesevilla) [inline doc/ref_arch/openstack/chapters/chapter02.rst:441]: @collivier error, it should be:
:ref:`ref_model/chapters/chapter05:cloud infrastructure hardware profiles features and requirements.`

File: doc/ref_arch/openstack/chapters/chapter02.rst [hunk 3]
```diff
@@ -1178,8 +1196,11 @@ Confidentiality and Integrity (source `RM 7.9.3 <../../../ref_model/chapters/cha
 Table 2-8: Reference Model Requirements: Confidentiality and Integrity
 Requirements
 
-Workload Security (source `RM 7.9.4 <../../../ref_model/chapters/chapter07.md#794-workload-security>`__)
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Workload Security
+^^^^^^^^^^^^^^^^^
+
+(source `RM 7.9.4
+<../../../ref_model/chapters/chapter07.md#794-workload-security>`__)
```
Comment (karinesevilla) [inline doc/ref_arch/openstack/chapters/chapter02.rst:1203]: @collivier error, It should be 
:ref:`ref_model/chapters/chapter07:workload security`

File: doc/ref_arch/openstack/chapters/chapter02.rst [hunk 4]
```diff
@@ -1048,8 +1064,10 @@ Platform and Access (source `RM 7.9.2 <../../../ref_model/chapters/chapter07.md#
 Table 2-7: Reference Model Requirements - Platform and Access
 Requirements
 
-Confidentiality and Integrity (source `RM 7.9.3 <../../../ref_model/chapters/chapter07.md#793-confidentiality-and-integrity>`__)
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Confidentiality and Integrity
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+(source :ref:`ref_arch/openstack/chapters/chapter02:confidentiality and integrity`)
```
Comment (karinesevilla) [inline doc/ref_arch/openstack/chapters/chapter02.rst:1070]: error, it should be:
(source :ref:`ref_arch/openstack/chapters/chapter07:confidentiality and integrity`)

File: doc/ref_arch/openstack/chapters/chapter02.rst [hunk 5]
```diff
@@ -1270,8 +1291,10 @@ Workload Security (source `RM 7.9.4 <../../../ref_model/chapters/chapter07.md#79
 
 Table 2-9: Reference Model Requirements - Workload Security Requirements
 
-Image Security (source `RM 7.9.5 <../../../ref_model/chapters/chapter07.md#795-image-security>`__)
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Image Security
+^^^^^^^^^^^^^^
+
+(source :ref:`ref_arch/openstack/chapters/chapter06:image security`)
```
Comment (karinesevilla) [inline doc/ref_arch/openstack/chapters/chapter02.rst:1297]: error:
(source :ref:`ref_arch/openstack/chapters/chapter07:image security`)

**Review by karinesevilla — APPROVED** [review]

**Review by walterkozlowski — APPROVED** [review]

**Review by ildikov — APPROVED** [review]

File: doc/ref_arch/openstack/chapters/chapter04.rst [hunk 6]
```diff
@@ -73,10 +72,7 @@ OpenStack Control Plane Servers (Control Nodes)
 -  BIOS Requirements
 
 For OpenStack control nodes we use the BIOS parameters for the basic
-profile defined in `Chapter 5.4 of the Reference
-Model
-<../../../ref_model/chapters/chapter05.md#54-cloud-infrastructure-
-hardware-profiles-features-and-requirements>`__.
+profile defined in :ref:`ref_model/chapters/chapter05:cloud infrastructure hardware profiles features and requirements.`.
```
Comment (pgoyal01) [inline doc/ref_arch/openstack/chapters/chapter04.rst:75]: Typo
```suggestion
profile defined in :ref:`ref_model/chapters/chapter05:cloud infrastructure hardware profiles features and requirements`.
```

**Review by pgoyal01 — APPROVED** [review]

**Pull request merged**